### DISCUSSION TITLE
Syntax error: Invalid syntax on Debian

### DISCUSSION THREAD

**Discussion by sk1badev** [FabCx/TradingView-Webhook-Bot#90] [Q&A] [answered]
Hello, please help me, I really ask. I did everything as in the instructions, I added all the necessary information for the telegram in the config. After the launch main.py the message appears: 
**"file"/root/TradingView-Webhook-Bot/TradingView-Webhook-Bot/main.py ", line 7
    <!DOCTYPE html>
    ^
SyntaxError: invalid syntax"**

I tried to install it on CentOS, it didn't work either, when I go to point 8, the following text appears:[https://imgur.com/2vByluZ](url)

**Comment by sk1badev** [comment, marked as answer]
Well, now this error has stopped appearing on CentOS, but now there is another problem with the script, I was looking for a solution to the problem on the Internet, I did not find it. I keep getting **OSError: [Errno 98]**

> (TradingView-Webhook-Bot) [root@ovz1 TradingView-Webhook-Bot]# python main.py
> Traceback (most recent call last):
>   File "main.py", line 46, in <module>
>     serve(app, host="0.0.0.0", port=80)
>   File "/root/TradingView-Webhook-Bot/TradingView-Webhook-Bot/lib64/python3.6/site-packages/waitress/__init__.py", line 13, in serve
>     server = _server(app, **kw)
>   File "/root/TradingView-Webhook-Bot/TradingView-Webhook-Bot/lib64/python3.6/site-packages/waitress/server.py", line 85, in create_server
>     sockinfo=sockinfo,
>   File "/root/TradingView-Webhook-Bot/TradingView-Webhook-Bot/lib64/python3.6/site-packages/waitress/server.py", line 244, in __init__
>     self.bind_server_socket()
>   File "/root/TradingView-Webhook-Bot/TradingView-Webhook-Bot/lib64/python3.6/site-packages/waitress/server.py", line 361, in bind_server_socket
>     self.bind(sockaddr)
>   File "/root/TradingView-Webhook-Bot/TradingView-Webhook-Bot/lib64/python3.6/site-packages/waitress/wasyncore.py", line 396, in bind
>     return self.socket.bind(addr)
> OSError: [Errno 98] Address already in use

**Reply by fabston** [reply]
Hello,

the port you’re trying to use (80) is already in use. Please try another one like 8443.
